modules/kobalt-plugin-api/src/main/kotlin/com/beust/kobalt/TestDirective.kt

```diff
@@ -11,6 +13,17 @@ class TestConfig(val project: Project) {
     fun jvmArgs(vararg arg: String) {
         project.testJvmArgs.addAll(arg)
     }
+
+    fun includes(vararg arg: String) {
+        project.testIncludes.apply {
+            clear()
+            addAll(arg)
+        }
+    }
+
+    fun excludes(vararg arg: String) {
+        project.testExcludes.addAll(arg)
```

Comment:
Not symmetric with the code above, probably needs a `clear()` as well? Let's make both codes similar.